imports/data/feeds.coffee

```diff
@@ -0,0 +1,40 @@
+Feeds = new Meteor.Collection(null)
+[
+  _id: "1"
+  tags: ["AH", "EDR"]
+  label: "English"
+  checked: true
+,
+  _id: "7"
+  tags: ["ESP"]
+  label: "Español"
+,
+  _id: "12"
+  tags: ["RUS"]
+  label: "Русский"
```

Comment:
I would add label's translation in en_US to every label, otherwise it may be hard to read those labels on the front-end.
Showing e.g. "Русский (RUS)" could also work, but "Русский (Russian)" or just "Russian" might be a better way to make them readable.